src/command_modules/azure-cli-feedback/azure/cli/command_modules/feedback/custom.py

```diff
@@ -4,61 +4,557 @@
 # --------------------------------------------------------------------------------------------
 
 from __future__ import print_function
-import sys
+import os
+import math
+import platform
+import datetime
+
+from collections import namedtuple
 
 from knack.log import get_logger
 from knack.prompting import prompt, NoTTYException
 from knack.util import CLIError
 
-from azure.cli.core.util import COMPONENT_PREFIX, get_installed_cli_distributions
-from azure.cli.core.telemetry import set_feedback
+from azure.cli.core.util import get_az_version_string
+from azure.cli.core.azlogging import UNKNOWN_COMMAND, CMD_LOG_LINE_PREFIX
+from azure.cli.core.util import open_page_in_browser
+import pyperclip
+
+_ONE_MIN_IN_SECS = 60
+
+_ONE_HR_IN_SECS = 3600
+
 
 logger = get_logger(__name__)
 
-MSG_RATE = '\nHow likely is it you would recommend our Azure CLI to a friend or colleague? [0 to 10]: '
-MSG_INTR = 'We appreciate your feedback! This survey is only two questions and should take less than a minute.'
-MSG_BAD = '\nWhat changes would we have to make for you to give us a higher rating? '
-MSG_GOOD = '\nWhat do we do really well? '
-MSG_EMIL = '\nIf you would like to join our insiders program and receive tips, tricks, and early access to new ' \
-           'features, let us know by leaving your email address (leave blank to skip): '
-MSG_THNK = '\nThanks for your feedback!'
+_MSG_THNK = 'Thanks for your feedback!'
+
+_GET_STARTED_URL = "aka.ms/azcli/get-started"
+_QUESTIONS_URL = "aka.ms/azcli/questions"
+_CLI_ISSUES_URL = "aka.ms/azcli/issues"
+_EXTENSIONS_ISSUES_URL = "aka.ms/azcli/ext/issues"
+
+_MSG_INTR = \
+    '\nWe appreciate your feedback!\n\n' \
+    'For more information on getting started, visit: {}\n' \
+    'If you have questions, visit our Stack Overflow page: {}\n'\
+    .format(_GET_STARTED_URL, _QUESTIONS_URL)
+
+_MSG_CMD_ISSUE = "\nEnter the number of the command you would like to create an issue for. " \
+                 "\nEnter {} to create a generic issue. Enter q to quit: "
+
+_MSG_ISSUE = "Would you like to create an issue? Enter Y or N: "
+
+_ISSUES_TEMPLATE_PREFIX = """
+
+BEGIN TEMPLATE
+===============
+**A browser has been opened to {} to create an issue.**
+**The issue template has been copied to your clipboard. You can also run `az feedback --verbose` to emit the output to stdout.**
+"""
+
+_ISSUES_TEMPLATE = """
+
+### **This autogenerated template contains command parameters. Please review and remove sensitive information, if necessary**
+
+## Describe the bug
+
+**Command Name**
+    `{command_name}`
+
+**Ran:**
+    `{executed_command}`
```

Comment:
It is safer to put sanitized values here and invite the user to put actual values than vice-versa. @JasonRShaver @yugangw-msft you know more about the risks here as they are largely the same as the risks involved with telemetry.